templates/ai/index.html

```diff
@@ -0,0 +1,544 @@
+                url="https://assets.ubuntu.com/v1/31c63bdf-onnx logo.png",
+                alt="ONNX",
+                width="348",
+                height="384",
+                hi_def=True,
+                loading="lazy",
+                attrs={"class": "p-logo-section__logo"}
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{
+              image(
+                url="https://assets.ubuntu.com/v1/ddbc4d23-kserve-logo.png",
+                alt="Kserve",
+                width="125",
+                height="384",
+                hi_def=True,
+                loading="lazy",
+                attrs={"class": "p-logo-section__logo"}
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{
+              image(
+                url="https://assets.ubuntu.com/v1/73accfaf-nvidia-triton-server-logo.png",
+                alt="Nvidia Triton Server",
+                width="249",
+                height="384",
+                hi_def=True,
+                loading="lazy",
+                attrs={"class": "p-logo-section__logo"}
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{
+              image(
+                url="https://assets.ubuntu.com/v1/797287f7-grafana-logo.png",
+                alt="Grafana",
+                width="173",
+                height="384",
+                hi_def=True,
+                loading="lazy",
+                attrs={"class": "p-logo-section__logo"}
+              ) | safe
+            }}
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>以 MLOps 方法（机器学习运维）将模型部署到实际生产环境</h2>
+      </div>
+      <div class="col">
+        <p>
+          使用 Canonical 的模块化 MLOps 解决方案，在任何规模的机器学习简化工作流程并自动化部署。 MLOps 使您能够高效地大规模部署和维护模型以进行生产。
+        </p>
+        <p>
+          由 Canonical 支持的 Kubeflow 发行版，能与不断发展的 AI 和数据工具生态系统相结合，为您的机器学习操作做到可扩展、可移植和可重制性。
+        </p>
+        <div class="p-cta-block">
+          <a href="/engage/mlops-guide" class="p-button--positive">MLOps指南</a>
+          <a href="/blog/mlops-challenges-cn" class="p-button">MLOps五大挑战</a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  {%- call(slot) vf_tiered_list(is_list_full_width_on_tablet=false) -%}
+  {%- if slot == 'title' -%}
+  <h2>开源 MLOps 工具选择</h2>
+  {%- endif -%}
+
+  {%- if slot == 'list_item_title_1' -%}
+  <h3 class="p-heading--5">Charmed Kubeflow</h3>
+  {%- endif -%}
+
+  {%- if slot == 'list_item_description_1' -%}
+  <p>
+    Charmed Kubeflow 是一款端到端的 MLOps 平台，旨在大规模运行 AI。它是 Canonical MLOps 的基础，并能与其他大数据和机器学习工具无缝集成。
+  </p>
+  <hr class="p-rule--muted u-hide--small" />
+  <p>
+    <a href="https://ubuntu.com/ai/what-is-kubeflow">什么是 Kubeflow&nbsp;&rsaquo;</a>
+  </p>
```

Comment:
Sorry, I should have been clearer when flagging this in separate sections, every instance of this design pattern should use p-cta-block. Basically if it's a link at the bottom of a block of content it should be a [cta-block](https://vanillaframework.io/docs/patterns/cta-block)
```suggestion
  <div class="p-cta-block">
    <a href="https://ubuntu.com/ai/what-is-kubeflow">什么是 Kubeflow&nbsp;&rsaquo;</a>
  </div>
```